cmd/owl2proto/owl2proto.go

```diff
@@ -383,9 +381,15 @@ func addDataProperties(output, rmk string, i int, preparedOntology ontology.Onto
 	// Create output for the data properties
 	for _, r := range dataProperties {
 		if r.Typ != "" && r.Value != "" {
-
 			var opts string = ""
 
+			// Get list of resource types for given  object
+			resourceTypeList := getResourceTypeList(preparedOntology.Resources[rmk], &preparedOntology)
+
+			// Get field number
+			resourceTypeList = append(resourceTypeList, r.Value)
+			fieldNumber := util.GetFieldNumber(resourceTypeList...)
```

Comment:
Looks good in general. I wonder whether we could make the consteitn field number vs. ++ a CLI option?